packages/next-app/src/pages/index.tsx

```diff
@@ -19,6 +28,9 @@ const HomePage: NextPage = () => {
       return;
     }
     setAssets(data.assets);
+
+    if (data.assets_aggregate.aggregate?.count) setHasMore(data.assets_aggregate.aggregate?.count > data.assets.length);
```

Comment:
I prefer to use
```
if(){
}
```

reference
https://github.com/airbnb/javascript#blocks

We haven't apply this rule to eslint, but let's set this as our coding rule.